inc/Engine/Optimization/LazyRenderContent/Frontend/Controller.php

```diff
@@ -145,10 +145,29 @@ public function add_hashes( $html ) {
 		$processor = wpm_apply_filters_typed( 'string', 'rocket_lrc_processor', 'dom' );
 
 		$this->processor->set_processor( $processor );
+		$this->processor->get_processor()->set_exclusions( $this->get_exclusions() );
 
 		return $this->processor->get_processor()->add_hashes( $html );
 	}
 
+	/**
+	 * Get the list of patterns to exclude from hash injection.
+	 *
+	 * @return string[]
+	 */
+	private function get_exclusions(): array {
+		/**
+		 * Filters the list of patterns to exclude from hash injection.
+		 *
+		 * @since 3.17
+		 *
+		 * @param string[] $exclusions The list of patterns to exclude from hash injection.
+		 */
+		$exclusions = wpm_apply_filters_typed( 'array', 'rocket_lrc_exclusions', [] );
```

Comment:
Starting from typed library version 1.2, we started the support of using array of strings
https://github.com/wp-media/apply-filters-typed/releases/tag/v1.2

What about this suggestion?

```suggestion
		$exclusions = wpm_apply_filters_typed( 'string[]', 'rocket_lrc_exclusions', [] );
```